### PR TITLE
Updating docs and hiding old flags for cert rotate command

### PR DESCRIPTION
**Pull request by jayvikramsharma1**
### :nut_and_bolt: Description: What code changed, and why?
We are supporting rotate cluster certificates flags in certificate rotation, so old flags to update individual service or individual Ips certificates will be deprecates and we are hiding those flags, and updating the docs for the same

we have added remove unreachable nodes flag in add and remove node commands, updating doc for the same,

upgrading PG version in doc

### :chains: Related Resources

https://chefio.atlassian.net/browse/CHEF-11744
https://chefio.atlassian.net/browse/CHEF-12275

### :+1: Definition of Done

Doc changes

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
Screenshot 2024-05-08 at 2 17 13 PM

### CODE REVIEW

**Review by punitmundra — CHANGES_REQUESTED** [review]
Please add the screenshot in the ticket with the existing flag and removed flag.

File: components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md [hunk 1]
```diff
@@ -149,11 +162,14 @@ You can mix and match different services to add nodes across various services.
 
 Once the command executes, it will add the supplied nodes to your automated setup. The changes might take a while.
 
+
```
Comment (shaik80) [inline components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md:165]: why are adding new lone here

**Review by shaik80 — APPROVED** [review]

**Review by punitmundra — APPROVED** [review]

File: components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md [hunk 2]
```diff
@@ -77,6 +77,19 @@ You can mix and match different services to add nodes across various services.
     ```sh
     chef-automate node add --automate-ips 10.1.2.23 --chef-server-ips 10.1.0.36,10.0.1.233  --postgresql-ips 10.0.1.42,10.0.1.54
     ```
+You can remove unreachable nodes while adding nodes
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md:80]: Add a full stop to end the line.

File: components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md [hunk 3]
```diff
@@ -77,6 +77,19 @@ You can mix and match different services to add nodes across various services.
...
+
+- To remove unreachable nodes while adding new nodes, pass the flag `--remove-unreachable-node` or `-r` like example below
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md:82]: Replace it with: - To remove unreachable nodes while adding new nodes, pass the flag `--remove-unreachable-node` or `-r` like the example below:

File: components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md [hunk 4]
```diff
@@ -149,11 +162,14 @@ You can mix and match different services to add nodes across various services.
 
 Once the command executes, it will add the supplied nodes to your automated setup. The changes might take a while.
 
+
 {{< note >}}
 
 - The new node will be configured with the certificates that are already configured in your HA setup.
 - If you had applied unique certificates per node, then the certificates of one of the nodes have been applied by default on the new nodes.
 - If you want to change the certificates for the new nodes, you can manually run the chef-automate cert-rotate [options] command.
+- Removing unreachable nodes while adding nodes is not supported in AWS mode of deployment, you have to use [remove node commands page](/automate/ha_remove_single_node_from_cluster/)
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md:171]: Replace it with: - Removing unreachable nodes while adding nodes is not supported in AWS mode of deployment; you have to use [remove node commands page](/automate/ha_remove_single_node_from_cluster/)

File: components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md [hunk 5]
```diff
@@ -63,6 +63,20 @@ The command requires some arguments to determine which types of nodes you want t
     chef-automate node remove --postgresql-ip "<postgresql-ip-address>"
     ```
 
+You can remove other unreachable nodes while removing a node
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md:66]: Add a full step

File: components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md [hunk 6]
```diff
@@ -63,6 +63,20 @@ The command requires some arguments to determine which types of nodes you want t
...
+
+- To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like example below
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md:68]: Replace it with: - To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like the example below

File: components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md [hunk 7]
```diff
@@ -107,4 +121,18 @@ The command requires some arguments to determine the type of node you want to re
     chef-automate node remove --postgresql-ip "<postgresql-ip-address>"
     ```
 
+You can remove other unreachable nodes while removing a node
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md:124]: add full stop

File: components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md [hunk 8]
```diff
@@ -107,4 +121,18 @@ The command requires some arguments to determine the type of node you want to re
...
+
+- To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like example below
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md:126]: - To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like the example below:

**Review by dishanktiwari2501 — APPROVED** [review]

**Review by AnkurMundhra — APPROVED** [review]
I can see that we are removing cert-rotate section for each service and node. If this is desired, then I am good.

**Pull request merged**